meteor-app/imports/ui/components/return-to-kiosk-button.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+
+function Kioskbutton() {
+  return (
+    <div>
+      <button>
+        <Link to="/">home</Link>
+      </button>
```

Comment:
It is better practice to use code for navigation, ie do a `history.push()`, because if gives you more control over the destination, as opposed to using a hard coded Link. If you are in kiosk mode, the home url is /kiosk, there is a method called `goHome()` that you should probably be using, and in fact this component should probably be called "HomeButton", as we need to put it in several places

Reply:
I'm looking into fixing this via you specifications now. Thanks for the feedback.